test/src/406-ducc-webhook-notifications/main

```diff
@@ -0,0 +1,139 @@
+cvmfs_test_name="Webhook notifications"
+cvmfs_test_autofs_on_startup=false
+cvmfs_test_suites="ducc"
+
+check_id_action() {
+    expected_id=$1
+    expected_action=$2
+
+    notification=$( tail -n 1 $notifications_file )
+    id=(${notification//|/ })
+
+    if [ ${id[0]} != ${expected_id} ]
+    then
+        echo 'The id of the action is not correct'
+        kill $pid
```

Comment:
Let's use a `trap` to stop the webhook.  You can search for `trap` in other integration tests and use it as a template